fastlane/Fastfile

```diff
@@ -53,14 +53,21 @@ private_lane :prepare_release_notes do |options|
   optional_preamble = options[:preamble]
   optional_postamble = options[:postamble] # Yes, it is a real word
 
-  release_notes = prs.map { |pr|
+  # Discard any PR which omits the release_notes tags
```

Comment:
We could do this with a regex instead, but personally I find those harder to read/understand